packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -11,47 +11,48 @@ const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;
 const { address: ERC20_ADDRESS } = MyERC20Full;
 const { address: NFT_ADDRESS } = MyERC721Full;
 
-// 100 gwei
-const GAS_PRICE = bigNumberify(`${1e11}`);
-
 const { ZERO, ONE } = constants;
+const SMALL_AMOUNT = bigNumberify(`${1e17}`); // 0.1 ethers
 
 describe("GnosisSafe", () => {
-  let gnosisSafe;
   let root;
-  let johnWallet;
-  let ephemeralWallet;
+  let gnosisSafeOwner;
+  let ephemeralAccount;
```

Comment:
I like these more explicit names rather than having to remember what role John serves in this example

Reply:
Resolving this since you gave it a thumbs up.